app/scripts/controllers/preferences.js

```diff
@@ -246,11 +269,13 @@ class PreferencesController {
    *
    */
   removeToken (rawAddress) {
-    const tokens = this.store.getState().tokens
+    const accountTokens = this.store.getState().accountTokens
+    const selectedAddress = this.store.getState().selectedAddress
+    const providerType = this.network.providerStore.getState().type
+    const updatedTokens = accountTokens[selectedAddress][providerType].filter(token => token.address !== rawAddress)
```

Comment:
Might want to normalize the `rawAddress`, to make sure we're using the same capitalization as the `accountTokens` object uses.  Probably also applies to `addToken`?

Reply:
The normalization of `rawAddress` wasn't in the code. I think it is because when you remove a token you select the address from the tokens list on the controller, that is already normalized when it was added with `addToken`, method that normalize the address, so it may not necessary I believe.